app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -961,7 +934,9 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
     }
 
     fun trackCreatorDetailsClicked(projectData: ProjectData) {
-        client.track(CREATOR_DETAILS_CLICKED, experimentProperties(projectData))
+        val props: HashMap<String, Any> = hashMapOf(CONTEXT_CTA.contextName to CREATOR_DETAILS.contextName)
```

Comment:
We are leaving all koala events as is and creating new methods for segment events and sending them alongside the old koala events.

Reply:
ok thanks

Reply:
@leighdouglas 
![image](https://user-images.githubusercontent.com/1075310/110827909-2f4e3e80-829f-11eb-9821-fe440c7eb71d.png)

Reply:
Great!